data/lib/core/container.lua

```diff
@@ -10,8 +10,8 @@ function Container.createLootItem(self, item)
 	local itemCount = 0
 	local randvalue = getLootRandom()
 	if randvalue < item.chance then
-		if ItemType(item):isStackable() then
-			itemCount = randvalue % item.countmax + 1
+		if ItemType(item.itemId):isStackable() then
```

Comment:
The field name for an item's id is ``itemid`` not ``itemId`` so it becomes ``item.itemid`` and not ``item.itemId``.

Reply:
we are using getLoot function to call this new table.
you can check this here:
https://github.com/otland/forgottenserver/blob/master/src/luascript.cpp#L12736
@raymondtfr